### PR TITLE
[NUI] Add null pointer check for Scrollbar

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
Add null pointer check for Scrollbar on ScrollableBase to prevent crash

### CODE REVIEW

**Review by elishateng — APPROVED** [review]
LGTM

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by huayongxu — APPROVED** [review]
lgtm.

**Review by AchoWang — APPROVED** [review]
Looks good to me

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 1]
```diff
@@ -329,25 +329,28 @@ public ScrollbarBase Scrollbar
             }
             set
             {
-                if (scrollBar)
+                if (value != null)
```
Comment (huiyueun) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs:332]: If this code is added, the ScrollBar can not set to null. Is this the correct behavior?

Comment (zhouleonlei) [reply src/Tizen.NUI.Components/Controls/ScrollableBase.cs]: Updated. Please check again.
Thanks

**Review by dongsug-song — APPROVED** [review]
LGTM

**Pull request merged**